src/ast/attachpoint_parser.cpp

```diff
@@ -325,16 +325,25 @@ AttachPointParser::State AttachPointParser::special_parser()
 
 AttachPointParser::State AttachPointParser::kprobe_parser(bool allow_offset)
 {
-  if (parts_.size() != 2)
+  if (parts_.size() != 2 && parts_.size() != 3)
```

Comment:
nit: maybe save off a local var e.g. `auto parts_size = parts_.size()`

Reply:
Yeah, that does look nicer.